pages/room/[id].js

```diff
@@ -39,6 +40,23 @@ export default function Room(props) {
     })
   }
 
+  //  For next sprint
```

Comment:
Can the commented code live in a stash until next sprint?

Reply:
might as well close this